test/replicator.spec.js

```diff
@@ -102,18 +102,13 @@ Object.keys(testAPIs).forEach((IPFS) => {
         assert.strictEqual(replicator.tasksStarted, 0) // ??
 
         replicator.on('load.end', async (replicatedLogs) => {
-          replicated++
-          assert.strictEqual(replicator.tasksStarted, replicated) // ??
+          // replicated++
+          // assert.strictEqual(replicator.tasksStarted, replicated) // ??
```

Comment:
TBD testing strategy here since there's no longer a 1:1 ration between replication tasks and replicated entries